ex/high_throughput_inference/mock_app.py

```diff
@@ -137,39 +98,40 @@ def run_model(self, model: bytes | str, batch: torch.Tensor):
             output_descriptors=[],
             custom_attributes=None,
         )
-        self.measure_time("build_request")
+        self.perf_timer.measure_time("build_request")
         request_bytes = MessageHandler.serialize_request(request)
-        self.measure_time("serialize_request")
-        with self._to_worker_fli.sendh(
-            timeout=None, stream_channel=self._to_worker_ch
-        ) as to_sendh:
+        self.perf_timer.measure_time("serialize_request")
+        self.perf_timer.measure_time("serialize_tensor")
```

Comment:
I think this line can be removed after putting `tobytes` back into the send call.

Reply:
Of course